e2e/cypress/integration/enterprise/guest_accounts/guest_invitation_ui_spec.js

```diff
@@ -315,4 +319,45 @@ describe('Guest Account - Guest User Invitation Flow', () => {
         // * Verify the content and message in next screen
         verifyInvitationError(email, 'Error: Email invitations are disabled.');
     });
+
+    it('MM-18047 Verify Guest User whitelisted domains', () => {
+        // #Configure a whitelisted domain
+        changeGuestFeatureSettings(true, true, 'example.com');
+
+        // # Login as sysadmin and visit to newly created team
+        cy.apiLogin('sysadmin');
+        cy.visit(`/${testTeam.name}`);
+
+        // # Invite a Guest by email
+        const email = `temp-${getRandomInt(9999).toString()}@mattermost.com`;
+        invitePeople(email, 1, email);
+
+        // * Verify the content and message in next screen
+        const expectedError = `Error: The following email addresses do not belong to an accepted domain: ${email}. Please contact your System Administrator for details.`;
+        verifyInvitationError(email, expectedError);
+
+        // # From System Console try to update email of guest user
+        cy.createNewUser().then((user) => {
+            // # Demote the user from member to guest
+            cy.demoteUser(user.id);
+
+            // # Navigate to System Console Users listing page
+            cy.visit('/admin_console/user_management/users');
+
+            // # Search for User by username and select the option to update email
+            cy.get('#searchUsers').should('be.visible').type(user.username);
+
+            // # Click on the option to update email
+            cy.getByTestId('userListRow').find('.MenuWrapper a').should('be.visible').click();
+            cy.getByText('Update Email').should('be.visible').click();
+
+            // * Update email outside whitelisted domain and verify error message
+            cy.getByTestId('resetEmailModal').should('be.visible').within(($el) => {
+                cy.wrap($el).getByTestId('resetEmailForm').should('be.visible').find('input').type(email);
```

Comment:
You can omit the `cy.wrap($el)`s here. With `within` you are already in the context of the parent element.

Reply:
Changes done. Thanks.